doc/cabal-project.rst

```diff
@@ -1536,7 +1539,7 @@ Advanced global configuration options
 -------------------------------------
 
 .. cfg-field:: write-ghc-environment-files: always, never, or ghc8.4.4+
-               --write-ghc-environment-files=policy
+               --write-ghc-environment-files=always\|never\|ghc8.4.4+
```

Comment:
Not sure why it's not simply `--write-ghc-environment-files=POLICY`, i.e. why inline the options that are listed just one line above.

Reply:
Do you want me to update the command output to use `POLICY`? This is what we have right now:

```
--write-ghc-environment-files=always|never|ghc8.4.4+
                                Whether to create a .ghc.environment file
                                after a successful build (v2-build only)
```